Snowflake.Data.Tests/UnitTests/SFSessionPropertyTest.cs

```diff
@@ -134,6 +135,74 @@ public static IEnumerable<TestCase> ConnectionStringTestCases()
                     { SFSessionProperty.ALLOWUNDERSCORESINHOST, defAllowUnderscoresInHost }
                 }
             };
+
+            var testCaseWithEmptyUserAndPassword = new TestCase()
```

Comment:
these tests are following the the pattern as it was written earlier. It is OK, but maybe we could write them this way: https://github.com/snowflakedb/snowflake-connector-net/blob/4d855888608e984be43f1005ce18a15aa55012de/Snowflake.Data.Tests/UnitTests/SFSessionPropertyTest.cs#L37 The problem with earlier approach was that it tested many properties at once even though we would like to test only one aspect (e. g. that the user and password can be empty). So my proposal would be to try to write it another way.

Reply:
The new tests are refactored using the new approach.